corehq/apps/tour/tours.py

```diff
@@ -0,0 +1,54 @@
+import collections
+from django.utils.safestring import mark_safe
+from django.utils.translation import ugettext_noop as _
+from corehq.apps.tour.utils import tour_is_enabled_for_user
+
+
+GuidedTourStep = collections.namedtuple('GuidedTourStep', ['element', 'title', 'content'])
+
+
+class StaticGuidedTour(object):
+
+    def __init__(self, slug, steps):
+        self.slug = slug
+        if not isinstance(steps, list):
+            raise ValueError("steps should be a list of GuidedTourStep")
+        self.steps = steps
+
+    def get_tour_data(self):
+        return {
+            'steps': map(lambda s: dict(s._asdict()), self.steps),
+        }
+
+    def is_enabled(self, user):
+        return tour_is_enabled_for_user(self.slug, user)
+
+
+SIMPLE_NEW_APP = StaticGuidedTour('simple_new_app', [
```

Comment:
(not actionable yet) might be worth thinking about whether we could represent these in a serialization format (like json or yaml) so that in theory we could define them dynamically, or at least not in python code.